src/Umbraco.Web/ImageCropperTemplateExtensions.cs

```diff
@@ -30,6 +30,9 @@ public static class ImageCropperTemplateExtensions
         /// </returns>
         public static string GetCropUrl(this IPublishedContent mediaItem, string cropAlias) => ImageCropperTemplateCoreExtensions.GetCropUrl(mediaItem, cropAlias, Current.ImageUrlGenerator);
 
+        public static string GetCropUrl(this MediaWithCrops mediaWithCrops, string cropAlias) => ImageCropperTemplateCoreExtensions.GetCropUrl(mediaWithCrops, cropAlias, Current.ImageUrlGenerator);
+
+        [Obsolete("This method does not get the crops or cache buster value from the media item.")]
```

Comment:
We might want to fix this method instead of making it obsolete, will get back to this later!

Reply:
This method could be useful to only get a crop from the local crops: `image.Content.GetCropUrl("Hero", image.LocalCrops)`, but lacks quite a few parameters and doesn't add the automatic cache buster value based on the media update date/time. It also doesn't align with the `UrlHelper` extension method: `Url.GetCropUrl(image.LocalCrops, "Hero")` (which has the same flaw BTW).

We should probably look into having all these extension methods aligned better and decide on how to best handle only using local crops: not something we need to block this PR for.

Reply:
Fixed in 2980ff9c54fe7c4bdf4f0c19a5672f148c3cd5d2 by forwarding this to the `GetCropUrl` extension method on `IPublishedContent` and adding flag to only use the local crops that are passed in 👍🏻